hbase-server/src/test/java/org/apache/hadoop/hbase/util/TestFSUtils.java

```diff
@@ -415,8 +417,9 @@ public void testSetStoragePolicyDefault() throws Exception {
    * Note: currently the default policy is set to defer to HDFS and this case is to verify the
    * logic, will need to remove the check if the default policy is changed
    */
-  private void verifyNoHDFSApiInvocationForDefaultPolicy() {
+  private void verifyNoHDFSApiInvocationForDefaultPolicy() throws URISyntaxException, IOException {
     FileSystem testFs = new AlwaysFailSetStoragePolicyFileSystem();
+    testFs.initialize(new URI("hdfs://localhost/"), conf);
```

Comment:
This line is required otherwise uri is null pointer and will fail hasPathCapability() API.

Reply:
Can we not use the 'hdfs' scheme here? It is not HDFS, actually...